.deploy/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+if [[ -z "${RAILS_MASTER_KEY}" ]]; then
```

Comment:
I'm under the impression that you check is false.
`if [[ -z "${RAILS_MASTER_KEY}" ]]; then`

returns true is the variable is empty.

Reply:
Indeed, your impression is right.

Reply:
A detail, but you could 've added `exit 1` if the argument is empty, and therefore avoid the `else`, which removes a level of indentation for the rest of the code.